### ISSUE TITLE
PushMessage init NPE

### ISSUE THREAD

**Issue by gvoltr** [urbanairship/android-library#19]
Greetings, I have few dozens of NPE crashes after updating to latest library version.

Log:
```
Fatal Exception: java.lang.RuntimeException: Unable to start receiver com.urbanairship.push.GcmPushReceiver: java.lang.NullPointerException
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:2431)
       at android.app.ActivityThread.access$1500(ActivityThread.java:141)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1332)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:5103)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:546)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:737)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.NullPointerException
       at com.urbanairship.push.PushMessage.<init>(PushMessage.java:238)
       at com.urbanairship.push.GcmPushReceiver.onReceive(GcmPushReceiver.java:61)
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:2424)
       at android.app.ActivityThread.access$1500(ActivityThread.java:141)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1332)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:5103)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:546)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:737)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by BrianBatchelder** [comment]
@gvoltr: could you provide us with the version number of the library version are you using? Thanks.

**Comment by rlepinski** [comment]
Fixed in 8.8.2
